shell/platform/darwin/ios/ios_external_view_embedder.h

```diff
@@ -0,0 +1,71 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_DARWIN_IOS_IOS_EXTERNAL_VIEW_EMBEDDER_H_
+#define FLUTTER_SHELL_PLATFORM_DARWIN_IOS_IOS_EXTERNAL_VIEW_EMBEDDER_H_
+
+#include "flutter/flow/embedded_views.h"
+#import "flutter/shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.h"
+
+namespace flutter {
+
+class IOSExternalViewEmbedder : public ExternalViewEmbedder {
+ public:
+  IOSExternalViewEmbedder(
+      FlutterPlatformViewsController* platform_views_controller,
+      std::shared_ptr<IOSContext> context);
+
+  // |ExternalViewEmbedder|
+  virtual ~IOSExternalViewEmbedder();
```

Comment:
Missing override.

Reply:
Done, always nice to have an extra safety net :-)